ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Get_Vendor_Files/tasks/main.yml

```diff
@@ -24,7 +24,7 @@
   run_once: true
 
 - name: Generate list of remote vendor_files
-  raw: dotgpg cat {{ item }}
+  command: dotgpg cat {{ item }}
```

Comment:
I had to switch this back because the raw command was adding a `\n` to the end of the output. It turned out that updating my gpg client fixed the problem that I was experiencing before